packages/web-components/fast-components/src/anchor/index.ts

```diff
@@ -36,6 +36,14 @@ export class Anchor extends FoundationAnchor {
         }
```

Comment:
I know this code isn't part of your change, but can we deprecate this class addition / removal behavior? It seems styling could just use the attribute, making the class un-necessary.